proposals/p0196.md

```diff
@@ -95,9 +95,7 @@ safety for the purposes of security. We think this proposal provides that
 foundation.
 
 Note: The decision by the core team included a request to clarify the wording on
-build modes as described in
-[chandlerc's post](https://forums.carbon-lang.dev/t/request-for-decision-language-level-safety-strategy/196/6)
-on the decision thread.
```

Comment:
are the forums archived anywhere?

Reply:
They generally weren't retained because of privacy concerns. It might still be worth retaining the broken link, as noted elsewhere.

Reply:
Yeah, I'd keep an annonated broken link here rather than losing text or obscuring that there was another place that's gone now.

Reply:
Okay. I've restored and annotated the broken links that had no replacement.